python/pyspark/sql/session.py

```diff
@@ -791,6 +792,20 @@ def udf(self) -> "UDFRegistration":
 
         return UDFRegistration(self)
 
+    @property
+    def udtf(self) -> "UDTFRegistration":
```

Comment:
We might want Spark Connect client to raise an exception for now:

https://github.com/apache/spark/blob/3bd45721f035fca14fabf59a75921cf245ea3b80/python/pyspark/sql/connect/session.py#L569-L572